bootstrap.go

```diff
@@ -116,13 +116,39 @@ const (
   		UNIQUE KEY name (name)
 		) ENGINE=InnoDB DEFAULT CHARSET=utf8 STATS_PERSISTENT=0 COMMENT='help topics';`
 
-	// CreateStatsMetaTable store's the meta of table statistics.
+	// CreateStatsMetaTable stores the meta of table statistics.
 	CreateStatsMetaTable = `CREATE TABLE if not exists mysql.stats_meta (
 		version bigint(64) unsigned NOT NULL,
 		table_id bigint(64) NOT NULL,
 		modify_count bigint(64) NOT NULL DEFAULT 0,
 		count bigint(64) unsigned NOT NULL DEFAULT 0,
-		index idx_ver(version)
+		index idx_ver(version),
+		unique index tbl(table_id)
+	);`
+
+	// CreateStatsColsTable stores the statistics of table columns.
+	CreateStatsColsTable = `CREATE TABLE if not exists mysql.stats_columns (
```

Comment:
How about name the table `stats_histgram`?
Because it contains both index and column.